stories/documentation/actions/button/button-group.stories.ts

```diff
@@ -41,7 +41,7 @@ function getTemplate(args: ButtonGroupStory): string {
 		<li class="button-group-item"><button type="button" class="button ${outlined} ${classes}">Bouton</button></li>
 		<li class="button-group-item">
 			<button type="button" class="button ${outlined} ${classes} mod-more">
-				<span class="lucca-icon icon-chevronSouth" aria-hidden="true"></span>
+				<span class="lucca-icon icon-arrowChevronBottom" aria-hidden="true"></span>
```

Comment:
it's weird, in the figma file this icon name is "icon-arrowChevronDown"

Reply:
we may need to harmonize between Bottom and Down

Reply:
On the code side, everything is top/bottom. I go back to Clément, well done!